scaling.adoc

```diff
@@ -214,26 +214,26 @@ or two orders of magnitude simpler than the Bitcoin protocol.
 ____
 
 The conclusion he makes is that sharding _might_ be technically
-possible, but it comes at the cost of tremendous complexity. Given
-that many users already find Bitcoin too complex and instead use
-centralized services, it's going to be hard to convince them to use
+possible, but it would come at the cost of tremendous complexity. Given
+that many users already find Bitcoin too complex and prefer to use
+centralized services instead, it's going to be hard to convince them to use
 something even more complex.
 
 ==== Inward scaling
 
-While horizontal and vertical scaling has worked out well historically
+While horizontal and vertical scaling have historically worked out well
 in centralized systems like databases and web servers, they don't seem
 to be suitable for a decentralized network like Bitcoin due to their
 centralizing effects.
 
-An approach that gets far too little appreciation is what we can call
```

Comment:
I wrote "what we can call" because I coined this term. It's not a generally accepted term (but it should be because it's so damn good).

If we can write "what I call", that'd be better. Can we write in first person here, referring to the author?